config.yaml

```diff
@@ -5,12 +5,12 @@ demo: &demo
       money:
         type: linear
         coeffs:
-          minimal: 10
+          minimal: "0.000010"
```

Comment:
That's bad. Please, parse the `Coin` values from `Number` values in Aeson instances.